### PR TITLE
Create using-terraform-with-google-cloud.md

### PR DESCRIPTION
**Pull request by csidyel**
Let me know if this should be created as a main or promoted pipeline. Any feedback is welcome.

### CODE REVIEW

File: docs/examples/using-terraform-with-google-cloud.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+## Store credentials in secrets
+
+- Store the Terraform Personal Access Token
+
+```bash
+sem create secret terraform-token \
+  -e TERRAFORM_TOKEN=<your-personal-token> 
+```  
+
```
Comment (bogyo210) [inline docs/examples/using-terraform-with-google-cloud.md:25]: This token was only needed for Digitalocean, the token is created on Digitalocean UI (web) to have a private api-key.
For gcloud only the ssh key and the gcloud.json is needed.
To be able to use the private key on gcloud, the public part must be added to the google-cloud-project 'Metadata/SSh keys'

Comment (csidyel) [reply docs/examples/using-terraform-with-google-cloud.md]: How does it look now? btw is it not necessary to run `ssh-keyscan` or `ssh-add` to make the connection work without any confirmation from our side?

**Review by csidyel — COMMENTED** [review]
@bogyo210 @AleksandarCole Can I merge this?

File: docs/examples/using-terraform-with-google-cloud.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+### Verify it works
+
+Push a new commit on any branch and open Semaphore to watch a new workflow run.
+If all goes well you'll see a `Passed` box next to your pipeline indicating
+the workflow finished successfully.
+
+### Next steps
+
+Congratulations! You have created a successful pipeline that
+communicates Terraform with Google Cloud.
```
Comment (bogyo210) [inline docs/examples/using-terraform-with-google-cloud.md:105]: Maybe a successful pipeline that applies a Terraform recipe to Google Cloud?

**Review by bogyo210 — COMMENTED** [review]
I think that people will find it interesting to see the actual recipe files, used by terraform. 
I know that was the hardest part to figure out.

**Review by csidyel — COMMENTED** [review]
@bogyo210 I updated the page as per our sync.

**Review by bogyo210 — APPROVED** [review]

**Pull request merged**